.github/workflows/pixi.toml

```diff
@@ -162,6 +162,12 @@ cwd = "../.."
 env = { CC = "ccache $CC", CXX = "ccache $CXX", FC = "ccache $FC" }
 description = "Build SciPy (default settings)"
 
+[feature.build-task.target.win-64.tasks.build]
+cmd = "spin build --setup-args=-Dblas=openblas --setup-args=-Dlapack=openblas --setup-args=-Duse-g77-abi=true && cp .github/workflows/.pixi/envs/build/Library/bin/openblas.dll build-install/Lib/site-packages/scipy/linalg/openblas.dll"
```

Comment:
This isn't the prettiest, but it works so let's blame it on Windows being odd:) I thought for a minute about replacing `.github/workflows/` with an env var like `$PWD`, to improve running this locally, but it doesn't matter much right now. We can always improve that later.